frameworks/helloworld/src/main/java/com/mesosphere/sdk/helloworld/scheduler/ReversePhasesCustomizer.java

```diff
@@ -10,8 +11,10 @@
  * in reverse order.
  */
 public class ReversePhasesCustomizer implements PlanCustomizer {
+
     @Override
     public Plan updatePlan(Plan plan) {
+        // Definitely don't reverse the uninstall plan!
```

Comment:
Comment is sorta redundant but ehh...

Maybe rename the class to `ReverseDeployPhasesCustomizer`?

Reply:
Oh. Comment is nonsense now. Good catch.